setup.py

```diff
@@ -60,6 +60,7 @@ def read_version():
                  'sagemaker==1.50.1', 'tensorflow<2.0', 'docker-compose', 'boto3==1.10.50',
                  'six==1.13.0', 'python-dateutil>=2.1,<2.8.1', 'botocore==1.13.50',
                  'requests-mock', 'awscli==1.16.314'],
-        'benchmark': ['click']
+        'benchmark': ['click'],
+        ':python_version=="3.6"': ['sagemaker-experiments==0.1.7']
```

Comment:
Would this need to be fixed to be >3.5 too?

Reply:
Just like in the MXNet/Pytorch containers?

Reply:
it will work for now since the only 3 version is 3.6. should be updated it for future revisions though.

Reply:
ill update tf once this pr is merged